test/spec/trackingModules/canvas/PopupMenuTracking.spec.js

```diff
@@ -0,0 +1,141 @@
+import {
+  bootstrapModeler,
+  getBpmnJS,
+  inject
+} from 'test/TestHelper';
+
+import {
+  query as domQuery
+} from 'min-dom';
+
+
+import { BpmnJSTracking } from 'src';
+import PopupMenuTracking from 'src/trackingModules/canvas/popupMenu';
+
+
+describe('PopupMenuTracking', function() {
+
+  const diagram = require('test/spec/simple.bpmn').default;
+  let bpmnJSTracking;
+
+  beforeEach(bootstrapModeler(diagram, {
+    additionalModules: [
+      BpmnJSTracking,
+      PopupMenuTracking
+    ]
+  }));
+
+  beforeEach(function() {
+    bpmnJSTracking = getBpmnJS().get('bpmnJSTracking');
+  });
+
+  beforeEach(inject(function(bpmnJSTracking) {
+    bpmnJSTracking.enable();
+  }));
+
+
+  describe('should subscribe', function() {
+
+    it('popupMenu.open', inject(function(eventBus) {
+
+      // given
+      const spy = sinon.spy(bpmnJSTracking, 'track');
+
+      // when
+      eventBus.fire('popupMenu.open');
```

Comment:
I'd rather expect the tests to be more integration-like, i.e. the test should use `popupMenu.open`.